packages/noya-renderer/src/components/SketchFileRenderer.tsx

```diff
@@ -451,11 +455,43 @@ export default memo(function SketchFileRenderer() {
     );
   }, [isEditingPath, page, state]);
 
+  const symbol = useMemo(() => {
+    if (interactionState.type !== 'insertingSymbol' || !interactionState.point)
+      return;
+
+    const symbol = {
+      ...getSymbols(state).find(
+        ({ do_objectID }) => do_objectID === interactionState.symbolID,
+      ),
+    } as Sketch.SymbolMaster;
+
+    const symbolInstance = produce(symbol, (draft) => {
+      if (!symbol || !draft.style) return;
+
+      draft.style.contextSettings = {
+        _class: 'graphicsContextSettings',
+        blendMode: 1,
+        opacity: 0.5,
+      };
+
+      draft.frame = {
+        ...symbol.frame,
+        x: interactionState.point!.x - symbol.frame.width / 2,
```

Comment:
I think we can avoid the `point!` if we assign it to a variable outside the `produce` callback. E.g. `const point = interactionState.point`. TypeScript "forgets" that we already checked for `!interactionState.point` once we're in a different function